lib/main_common.pm

```diff
@@ -798,7 +798,7 @@ sub load_inst_tests {
     if (get_var('DUD_ADDONS') && is_sle('<15')) {
         loadtest "installation/dud_addon";
     }
-    loadtest 'installation/accept_license' if has_product_selection;
+    loadtest 'installation/accept_license' if has_license_to_accept;
```

Comment:
the accept license is scheduled for both products. is that how it should be? i think someone had mentioned that the accept license is handling in the welcome module for sle12. Can you double check?

Reply:
yes, it is like that, so now instead of doing it in the `welcome` module we do it always in `accept_license` module regardless of the product, so I have combined the conditions and adjusted `next` button.